READMEs/datatokens-flow.md

```diff
@@ -68,14 +81,41 @@ from ocean_lib.web3_internal.wallet import Wallet
 private_key = os.getenv('TEST_PRIVATE_KEY1')
 config = ExampleConfig.get_config()
 ocean = Ocean(config)
+print(f"config.network_url = '{config.network_url}'")
+print(f"config.block_confirmations = {config.block_confirmations.value}")
 
-print("create wallet: begin")
+print("Create wallet: begin")
 wallet = Wallet(ocean.web3, private_key, config.block_confirmations, config.transaction_timeout)
-print(f"create wallet: done. Its address is {wallet.address}")
-
-print("create datatoken: begin.")
-datatoken = ocean.create_data_token("Dataset name", "dtsymbol", from_wallet=wallet)
-print(f"created datatoken: done. Its address is {datatoken.address}")
+print(f"Create wallet: done. Its address is {wallet.address}")
+
+print("Create ERC721 token: begin.")
+erc721_token = ocean.create_nft_token(name="Dataset name", symbol="dtsymbol", from_wallet=wallet)
+print(f"Created ERC721 token: done. Its address is {erc721_token.address}")
+print(f"ERC721 token name: {erc721_token.token_name()}")
+print(f"ERC721 token symbol: {erc721_token.symbol()}")
+
+# Create ERC20 token related to the above NFT.
+from ocean_lib.models.models_structures import ErcCreateData
+from ocean_lib.web3_internal.constants import ZERO_ADDRESS
+from ocean_lib.web3_internal.currency import to_wei
+
+print("Create ERC20 token: begin.")
+cap = to_wei(10)
+erc20_data = ErcCreateData(
+        template_index=1, # default value 
+        strings=["ERC20DT1", "ERC20DT1Symbol"], # name & symbol for ERC20 token
+        addresses=[
+            wallet.address, # minter address
+            wallet.address, # fee manager for this ERC20 token
+            wallet.address, # publishing Market Address
+            ZERO_ADDRESS, # publishing Market Fee Token
+        ],
+        uints=[cap, 0],
+        bytess=[b""])
+erc20_token = erc721_token.create_erc20_token(erc20_data=erc20_data, from_wallet=wallet)
```

Comment:
please add an empty line before this one, to better delimit the steps. Please also move the closing `)` from erc create data to align correctly.

Reply:
and the entire content (arguments) is 1-unit misaligned, so please fix that as well